source/codegen/metadata/nidmm/enums.py

```diff
@@ -237,6 +237,18 @@
             }
         ]
     },
+    'FrequencyVoltageRange': {
```

Comment:
This is same as `Range` enum on line 508.

![image](https://user-images.githubusercontent.com/7554740/117243856-9e868000-ae55-11eb-9254-16f9cda7182b.png)

Reply:
But it seems to have one value different, AUTO_RANGE_ONCE is not defined for FREQ_VOLTAGE_RANGE

Reply:
Based on documentation, [ATTR_RANGE](https://zone.ni.com/reference/en-XX/help/370384V-01/dmmcref/canidmm_attr_range/) and [ATTR_FREQ_VOLTAGE_RANGE](https://zone.ni.com/reference/en-XX/help/370384V-01/dmmcref/canidmm_attr_freq_voltage_range/) attributes and even values seem to have bit different meaning as well. But the header file seems to leverage same macros for both of them. We can quickly check with Alex if he prefers same enum or different enum over email before completing this PR.

Reply:
Okay, I will check Alex today and update here in the PR accordingly.